docs/rfcs/014-born_digital_workflow/README.md

```diff
@@ -0,0 +1,176 @@
+# RFC 014: Born digital workflow
+
+**Last updated: 10 June 2019.**
+
+## Background
+
+As part of the move to our new [storage service](https://github.com/wellcometrust/platform/tree/master/docs/rfcs/002-archival_storage), we will be adopting [Archivematica](https://www.archivematica.org) to manage the preservation workflow for born-digital archives. This is analogous to our use of [Goobi](https://www.intranda.com/en/digiverso/goobi/goobi-overview/) for managing digitisation workflows.
+
+This document describes an initial workflow that will allow us to start using Archivematica as soon as possible. This means that, for now, we will continue to perform some parts of the workflow manually, outside of Archivematica.
+
+We will be actively contributing to the development of Archivematica, including ensuring that it better addresses our needs around appraisal and cataloguing. This will mean we are able to streamline the workflow over time, as the amount of born-digital material increases.
+
+## Proposed solution
+
+### Principles
+
+There are three key principles underlying this initial workflow:
+
+- Appraisal and selection will remain manual until the functionality in Archivematica improves
+- Cataloguing will be done in Calm and file-level description will not be part of the catalogue record
+- All processing steps within Archivematica will be fully automated and require no intervention
```

Comment:
This concerns me slightly if it means that we won't have the skills or understanding to know what has happened/gone wrong.

Reply:
Agree. Automating steps is good for efficiency but only if the whole team have a full understanding of exactly what is happening at each step, so that they can troubleshoot when things go wrong and have confidence in the integrity and authenticity of the results when things work well. If this means more manual interventions in these early stages of implementation, then so be it.